src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1709,6 +1711,8 @@ public static LivingEntity getLivingEntity(ModElement modElement, Random random,
 		livingEntity.modelHeight = 1.3;
 		livingEntity.mountedYOffset = -3.1;
 		livingEntity.modelShadowSize = 1.8;
+		for(int i = 0; i < 7; i++)
+			livingEntity.spawnPercentageRaid[i] = 5;

```

Comment:
It is not good that all fields have the same value for the test case. Consider using e.g. (i+1) or something like that

Reply:
Fixed